### PR TITLE
Verify that Vale actually works as intended

### PR DESCRIPTION
**Pull request by amotl**
Hi there,

this is an attempt to verify a file with specific "bad-style" content with the Vale prose linter in order to check if it will work as intended.

With kind regards,
Andreas.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -6,6 +6,9 @@ Changes
 Unreleased
 ==========
 
+- Add a quick ``make test`` target for validating that the
```
Comment (seut) [inline CHANGES.rst:9]: As the `make test` is only interested internally and not from an outside view it should not be mentioned here in my opinion.

Comment (amotl) [reply CHANGES.rst]: I've removed that comment from the CHANGES.rst file.

**Review by mechanomi — CHANGES_REQUESTED** [review]
thanks for this!

I am requesting changes because I don't think the Vale config should be duplicated under the test directory. this file is likely to see (relatively) significant and frequent updates over time and this duplication would mean that we have to update the config in two places, which violates the DRY principal

is there a reason you duplicated the file? can it be avoided?

**Comment by amotl** [comment]
> Is there a reason you duplicated the `vale.ini` file?

Sure. I need a deterministic set of settings for testing the outcome. It is common practice within different test suites to have a dedicated `test.ini` that can be adjusted individually and independently from the main `application.ini`.

**Comment by mechanomi** [comment]
@amotl in this instance, the Vale config is what is being tested. so creating a duplicate that is "known to work" renders the tests useless (except for demonstrating that the Vale binary itself is behaving as expected)

by using the actual Vale config for the test, we are verifying that the actual config we use is producing errors as expected, which seems valuable/useful

note that at the moment, our Vale config is very simple. but over time, the plan was to expand this work and really start to develop a config that reflects our style guide

**Comment by amotl** [comment]
The tests can well be extended to validate the production vale.ini using different test cases. My primary purpose was to create a harness for #43. As we now have this in place, it can evolve further.

**Comment by mechanomi** [comment]
@amotl for the purposes of a harness, in the manner you suggest, the Vale config should be reduced to the simplest possible thing that could work for that purpose

with that in mind:

- L2 can probably be deleted

- L5 can possibly be reduced so that we're using one, or maybe zero, "based on" styles. it depends on what you have in mind for this. so I'll let you decide whether this needs changing

- L7 onwards can be deleted. these are overrides and are specific to our use of Vale. for the purposes of a harness, we can just use the stock config

**Comment by amotl** [comment]
Dear Naomi,

I've tried to address your request as good as possible.

With kind regards,
Andreas.

**Review by mechanomi — APPROVED** [review]

**Pull request merged**